libsplinter/src/admin/store/diesel/migrations/postgres/migrations/2020-08-18-213009_admin_service_store/up.sql

```diff
@@ -113,6 +113,5 @@ CREATE TABLE IF NOT EXISTS circuit_member (
 CREATE TABLE IF NOT EXISTS node_endpoint (
     node_id                TEXT NOT NULL,
     endpoint               TEXT NOT NULL,
-    PRIMARY KEY (node_id, endpoint),
```

Comment:
Typo in commit message: "... node_id is not unique in  and the circuit_member ...". Also s/delete/deleted/ at the end.

Reply:
fixed